### ISSUE TITLE
No English language

### ISSUE THREAD

**Issue by Infiexe** [HidasMato/role_cards_creatures#1]
The generator is awesome. My only complaint is that it isn't in English and using it with Google Translate is a pain. It'd be neat to finally generate cards that resemble originals, especially with customizable text allowing for erratas

**Comment by HidasMato** [comment]
Hi, learn Russian, it may come in handy soon :D
(sorry)

I don't know English well, so I won't localize the generator.

No, seriously, Russian is an international language, like English, like Chinese, like Spanish, like many other. I can only translate it into Chinese on my own, but I won't. Plus, I made generators to impress my man, now it's my husband and I don't need to impress him anymore :D

The code for the generator is freely available, so you can translate the files if you want. Or you could find someone else proficient in javascript who would do it for you

Instructions if you don't know how to use node js
Download the files in the main branch 
Download node to your computer <a>https://nodejs.org/en/download/</a>
Run the console commands in main folder:
<pre>
npm i
npm start
</pre>

You will have a page on the Internet at the localhost address. 
Then you can change the Russian text in the files (You most likely need the file "role_cards_creatures /src /components /CreatureMain /Creatureman.tsx")
All changes should be immediately displayed in the browser (perhaps after saving the changes, I don't know, VSC automatically saves everything)
You can share these files with other people.
Or do as I have on github using github pages instruction: <a>https://github.com/gitname/react-gh-pages</a>
*You only need to change the addresses to the github repository

Translated with the help of Yandex Translator

**Comment by HidasMato** [comment]
And by the way, I advise you to write more detailed messages to people who do not speak your language. Because I didn't actually understand the phrase "It'd be neat to finally generate cards that resemble originals, especially with customizable text allowing for erratas". The translator translates this as some kind of nonsense 🙃


Translated with the help of Yandex Translator

**Comment by Infiexe** [comment]
Thank you for your response and suggestions!
What I meant was:
Your generator is the only one I know that creates cards that look like the original ones from Wizards of the Coast. It would be great to use it to generate cards in different languages. From what I know, your generator also lets you use custom card text, which is very useful when updates or errata come out. I understand that's not possible, though, and I appreciate your reply.
Regardless of whether I would like to learn Russian or not, I can't get my 10+ RPG players to do the same :c 
So I guess I will try to learn how to translate your generator myself as you instructed.

BTW, deepl.com translator might be a little bit better than Yandex translator

**Comment by HidasMato** [comment]
Hi

My previous answer now seems kind of rude to me, damn translators
I meant that I don't want to translate it into English <b>MYSELF</b>, because even in Russian it was difficult for me to find a name for the settings, not to mention English, which is much worse for me.

If you don't want to understand the code, you can provide me with a text file with the translation of some lines and I will make a separate repository with the English version.

The file is something like:
Russian text - English text

It seems to be easier for everyone

**Comment by Infiexe** [comment]
I've created an "alpha version" of the English translation, which I used locally to create beautiful cards. Since your wonderful generator allows to customize so many settings, I was able to create cards that are virtually indistinguishable from the original. The only feature I miss adding a background image/texture on the back of the card but that's nothing Photoshop can't solve :) 

I also wasn't able to add my own fonts, no matter if they were .TTF or .OTF, so I temporarily hardcoded them into my local generator copy. Anyway, the code was easy to understand, and I was able to make these kind of tweaks myself without much programming knowledge.

I agree that finding names for some of the settings was difficult, even given the fact I have some experience with English UX Writing. My "alpha" translation is enough for my purposes, but it's too crude to be presented to the world. As the generator serves me well, I don't think I'll find the motivation to improve the translation, but never say never :)
